.github/workflows/macosx.yml

```
@@ -21,6 +21,7 @@ jobs:
           CC: clang
           CFLAGS: "-fprofile-instr-generate -fcoverage-mapping"
           LDFLAGS: "-fprofile-instr-generate -fcoverage-mapping"
+          PKG_CONFIG_PATH: "/usr/local/opt/libxml2/lib/pkgconfig"
```

Apple silicon brew uses `/opt/homebrew` as prefix:
```suggestion
          PKG_CONFIG_PATH: "$(brew --prefix)/opt/libxml2/lib/pkgconfig"
```

---

I confirm that on my M1 mac the prefix is "/opt/homebrew".